react-payments-app/src/components/common/Modal/ModalRow.jsx

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useState } from 'react';
+
+const ModalRow = ({ data, onClick }) => {
+  const [isClicked, setIsClicked] = useState(false);
+  const [selectedId, setSelectedId] = useState(null);
+
+  const handleCompanySelected = (e) => {
+    const selectedId = e.target.parentNode.id;
+    setSelectedId(selectedId);
+    setIsClicked(!isClicked);
+  };
+
+  useEffect(() => {
+    onClick(isClicked, selectedId);
+  }, [isClicked, selectedId]);
+
+  return (
+    <div className='flex-center'>
+      {Array.isArray(data)
+        ? data?.map((company) => (
+            <div
+              key={company.id.toString()}
```

Comment:
```suggestion
              key={company.id}
```

key값은 string , number 모두 할당 가능하기 때문에, 굳이 number를 string으로 변환할 필요는 없을 것 같아요

Reply:
[7437f65](https://github.com/next-step/react-payments/pull/84/commits/7437f65585510426c6e6f681aff92e247cb9c1b3)